locales/ru.json

```
@@ -55,5 +55,105 @@
       "title": "Проекты",
       "help": "Хочу помочь"
     }
+  },
+  "ZakatPage": {
```

we try to make the keys follow the page in the naming
```suggestion
  "zakat": {
```